Source/ablastr/fields/MultiFabRegister.cpp

```diff
@@ -419,29 +415,45 @@ namespace ablastr::fields
     MultiLevelScalarField
     MultiFabRegister::internal_get_mr_levels (
         std::string const & name,
-        int finest_level
+        int finest_level,
+        bool skip_level_0
     )
     {
         MultiLevelScalarField field_on_level;
         field_on_level.reserve(finest_level+1);
         for (int lvl = 0; lvl <= finest_level; lvl++)
         {
-            field_on_level.push_back(internal_get(name, lvl));
+            if (lvl == 0 && skip_level_0)
+            {
+                field_on_level.push_back(nullptr);
```

Comment:
Is pushing back a `nullptr` the clean approach or not pushing back anything (`skip` might imply either).
Then on the other hand, I assume if we move the level indexing on `skip` that can cause a lot of confusion, too (one argument to not use a `Vector` for the list of pointers.)

Thus, ok for me to keep as you suggest for now.